srcpkgs/lc0/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'lc0'
+pkgname=lc0
+version=0.27.0
+revision=1
+wrksrc="lc0-${version}"
+build_style=meson
+configure_args="-Dgtest=false --buildtype=release"
+meson_builddir="build/release"
```

Comment:
Why force release?

What's special about `build/release`?

Reply:
I guess nothing, it works fine without specifying it.